src/test/java/com/box/sdk/BoxAPIConnectionTest.java

```diff
@@ -569,48 +569,65 @@ public void getLowerScopedTokenRefreshesTheTokenIfNeededbyCallingGetAccessToken(
         List<String> scopes = new ArrayList<String>();
         scopes.add("DummyScope");
         String resource = "";
+        String sharedLink = null;
 
         when(api.getTokenURL()).thenReturn("https://api.box.com/oauth2/token");
-        when(api.getLowerScopedToken(scopes, resource)).thenCallRealMethod();
+        when(api.getLowerScopedToken(scopes, resource, sharedLink)).thenCallRealMethod();
```

Comment:
will probably have to break this out into two tests - one for the original method and one for the new overloaded method